htdocs/comm/propal.php

```diff
@@ -293,7 +293,8 @@
 					$object->note = GETPOST('note');
 					$object->statut = 0;
 
-					$id = $object->create_from($user);
+					// the create is done below and further more the existing create_from function is quite hilarating
+					//$id = $object->create_from($user);
```

Comment:
the $object already contains fetched original lines and they are save in database on the call of the create function some lines below.
the create_from function just fill a $this->products array (which is never used) and call the create function.
so the create is done twice, there is a error message for duplicate record in db, but the propale has in fact being copied (with the first call of the create) but the user dont even know it because of the error of the second create call that make dolibarr stay on the same sceen.